input/pagecontent/StructureDefinition-datax-measurereport-deqm-intro.md

```diff
@@ -1,24 +1,30 @@
 {% assign id = {{include.id}} %}
-
-
+<!--Begin Generated Intro Tag (DO NOT REMOVE)-->
+### Mandatory Data Elements and Terminology
 The following data-elements are mandatory (i.e data MUST be present).
 
-**Each {{site.data.structuredefinitions.[id].type}} must have:**
-
-1. Whether is a [snapshot] or [incremental] update*
-1. A status
-1. measure report type hardcoded to "data-collection"
-1. The reference to the [Measure]({{site.data.fhir.path}}measure.html)
-1. A link to the subject the report is about
-1. The date the report was generated
-1. The organization that reported the data
-1. The measurement period
-
-Each {{site.data.structuredefinitions.[id].type}} *should* have ([Must Support]):
-
-1. References to other resources involved in producing the report
-1. Specify the software systems used by the reporting system using the CQFM [Software System]({{site.data.fhir.cqfm}}StructureDefinition-cqfm-softwaresystem.html) Extension
-1. A reporter.group
+**Each MeasureReport Must Have:**
+1. updatetype: Optional Extensions Element
+2. status: complete \| pending \| error
+3. type: individual \| subject-list \| summary \| data-collection
```

Comment:
This may need further discussion. It's been brought to my attention that rather than displaying the fixedCode value, it's the short description that needs to be updated.